doc/VMSpecificationSingleNode

```diff
@@ -0,0 +1,15 @@
+### VM Specifications - Single-Node VM deployment
```

Comment:
This newly created file needs to be linked from an existing file.

Reply:
yes, this file will be linked in build creation as well as deploy VM to follow as pre-requisite to setup VM as per recommended specification